src/SidebarBackButton/docs/index.story.js

```diff
@@ -0,0 +1,109 @@
+import {
+  header,
+  tabs,
+  tab,
+  description,
+  importExample,
+  title,
+  columns,
+  divider,
+  code as baseCode,
+  playground,
+  api,
+  testkit,
+} from 'wix-storybook-utils/Sections';
+
+import { storySettings } from '../test/storySettings';
+import allComponents from '../../../stories/utils/allComponents';
+import SidebarBackButton from '..';
+import * as examples from './examples';
+
+const code = config => baseCode({ components: allComponents, ...config });
+
+export default {
+  category: storySettings.category,
+  storyName: 'SidebarBackButton',
+
+  component: SidebarBackButton,
+  componentPath: '..',
+
+  componentProps: {
+    buttonText: 'Go back',
```

Comment:
Should be adjusted to:
```suggestion
    children: 'Go back',
```